soccer/src/soccer/planning/planner/plan_request.cpp

```diff
@@ -65,14 +65,21 @@ void fill_obstacles(const PlanRequest& in, rj_geometry::ShapeSet* out_static,
         out_static->add(std::make_shared<rj_geometry::Circle>(obs_center, obs_radius));
     }
 
+    // Adding ball as a static obstacle (because dynamic obstacles are not working)
+    // Only added when STOP state is enabled
+    double ball_vel_scaled = in.world_state->ball.velocity.mag() * .1;
+    double radius = kBallRadius + in.min_dist_from_ball * (1.0 + ball_vel_scaled);
```

Comment:
can you leave a comment explaining this math? are you sure it's equivalent to the math used in fill_robot_obstacle? 

I know we add a min_dist, but we should start with that min_dist and then inflate+offset using the same math as fill_robot_obstacle (you can honestly just call that function), this ensures we stay min_dist away from a moving target by adding extra buffer proportional to velocity.

Reply:
The math I was using previously was an equivalent version. However, I changed it to use the same math as the other function.